src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/SpringBootContractBaseTest.java

```diff
@@ -26,13 +26,30 @@
     public static final String PACT_TEST_SCOPES_VALUE = "openid profile roles";
     public static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
     public static final String AUTH_TOKEN = "Bearer someAuthorizationToken";
+<<<<<<< HEAD
     public static final String SERVICE_AUTH_TOKEN = "Bearer someServiceAuthorizationToken";
+<<<<<<< HEAD
+=======
+=======
+>>>>>>> 6e477b6c078906bb1812d0fd38917be42e32989d
+>>>>>>> 7275f41d0b8e5ccdecd242b16d1a8097fec3e1e6
+    public static final String EXPERIMENTAL = "true";
 
 
     public HttpHeaders getHttpHeaders() {
         HttpHeaders headers = new HttpHeaders();
+<<<<<<< HEAD
```

Comment:
looks like some merge issues ->
<<<<<<< HEAD
=======
=======

should not be there